portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,14 +26,33 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String[] wordArray = {"Hey!", "Check", "here!"};
+    /**
     String toJson = convertToJson(wordArray);
-
     response.setContentType("application/json;");
     response.getWriter().println(toJson);
+    */
   }
-  private String convertToJson(String[] wordArray) {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String getMessage = getParameter(request, "comment", "");
+    String submissionResponse = "Submitted! Thank you " + getParameter(request, "name", "");
```

Comment:
Same as the above comment about constant. The same probably also applies to the "name" constant.